resources/definitions/fdmprinter.def.json

```diff
@@ -4414,6 +4414,19 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
+                "prime_tower_purge_volume":
+                {
+                      "label": "Prime Tower Purge Volume",
+                      "description": "Amount of filament to be purged when wiping on prime tower. This is useful for compensating the filament lost by oozing during the inactivity of the nozzle.",
+                      "type": "float",
+                      "enabled": "resolveOrValue('prime_tower_enable')",
```

Comment:
The description tells us that the purge happens when wiping, so the setting should only be enabled when wiping is enabled.